docs/farming-&-staking/staking/staking.md

```diff
@@ -2,6 +2,7 @@
 title: Staking guide
 sidebar_position: 2
 description: Staking guide
+slug: /domains/nominator
```

Comment:
Maybe something like

```suggestion
slug /staking/nominate
```

or 
```suggestion
slug: /domains/staking-guide
```

Wdyt?

Reply:
I believe `staking` is appropriate here. I am omitting `-guide` since we are not including it in any other path.